Mod/Classes/Patched/MyMapScene.cs

```diff
@@ -0,0 +1,70 @@
+using TowerFall;
```

Comment:
You have a mix of spaces and tabs throughout this file. You should be using all spaces.

Reply:
I was afraid of calling base because of the base.add(button). I thought I needed my stuff to come before that.